app/models/user.rb

```diff
@@ -76,6 +76,10 @@ class User < ActiveRecord::Base
 
   has_many :reports
 
+  has_many :authorizations, class_name: 'OpenidConnect::Authorization'
+  has_many :o_auth_applications, through: :authorizations, class_name: 'OpenidConnect::OAuthApplication'
+  has_many :o_auth_access_tokens, through: :authorizations, class_name: 'OpenidConnect::OAuthAccessToken'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.